azkaban-common/src/main/java/azkaban/utils/SystemMemoryInfo.java

```diff
@@ -22,34 +17,46 @@
  * All the memory size used in this function is in KB
  */
 public class SystemMemoryInfo {
-  private static final Logger logger = Logger.getLogger(SystemMemoryInfo.class);
+  private static final org.slf4j.Logger logger = LoggerFactory.getLogger(SystemMemoryInfo.class);
```

Comment:
We already have both in place. I think it will be good to require new code to use slf4j. What do you think?